geometric/nifty.py

```diff
@@ -141,6 +141,8 @@ def emit(self, record):
 kj2au        = 1.0 / au2kj
 grad_au2gmx  = 49614.75258920567   # Previous value: 49614.75960959161
 grad_gmx2au  = 1.0 / grad_au2gmx
+au2ev        = 27.211386245988
+ev2au        = 1.0 / au2ev
 au2evang     = 51.422067476325886  # Previous value: 51.42209166566339
 evang2au     = 1.0 / au2evang
 c_lightspeed = 299792458.
```

Comment:
Can we use a units package like `openff-units` instead of updating the values manually?

Reply:
At the moment I'd rather not introduce a dependency on another package, but I will keep this in mind for the future.